misk/src/main/kotlin/misk/flags/memory/InMemoryDynamicFlagStore.kt

```diff
@@ -0,0 +1,42 @@
+package misk.flags.memory
+
+import misk.flags.DynamicFlagStore
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.TimeUnit
+import javax.inject.Singleton
+
+/** In memory implementation of a [DynamicFlagStore], suitable for use in testing */
+@Singleton
+class InMemoryDynamicFlagStore : DynamicFlagStore {
+    private val _booleanFlags = ConcurrentHashMap<String, InMemoryBooleanFlag>()
```

Comment:
why _ prefix?

Reply:
Because there are public vals that expose the various maps as immutable maps, and it seems like using a prefix for private vals that are exposed as modified public vals seems like a Kotlin convention.